bindings/azure/eventhubs/eventhubs.go

```diff
@@ -198,10 +198,11 @@ func (a *AzureEventHubs) RegisterPartitionedEventProcessor(handler func(*binding
 		return err
 	}
 
-	callback := func(c context.Context, event *eventhub.Event) error {
+	callback := func(c context.Context, event *eventhub.Event, bind *bindings.InvokeResponse) error {
```

Comment:
Is this change supposed to be part of this PR?

Reply:
no it's not, I am not sure how it even got attacked to this

Reply:
Thanks for catching that. Looks like my fork had that commit and probably just got pushed along with ok-to-test commit, just reverted it.